index.html

```diff
@@ -0,0 +1,390 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>David Underwood</title>
+  </head>
+  <body>
+    <header>
+      <a href="/" class="header-logo">
+        <h3>Welcome</h3>
+      </a>
+      <nav>
+        <img class="union" src="Union.png" alt="menu link" />
+      </nav>
+    </header>
+
+    <div class="greet">
+      <div class="top-row-one">
+        <h2 class="porto">Portoflio</h2>
+        <h3 class="about-top">About</h3>
+        <h4 class="con">Contact</h4>
+      </div>
+      <div class="text-box">
+        <h1 class="hi-there">Hey there. I’m <br />David</h1>
+        <h2 class="hi-there2">
+          I’m a software <br />
+          developer
+        </h2>
+      </div>
+      <div class="greeting2">
+        <h1 class="header-title">Hey there. I’m David.</h1>
+        <h1>
+          <span>I’m a software developer</span>
+        </h1>
+      </div>
+      <p class="paragraph1">
+        I can help you build a product, <br />
+        feature or website Look through <br />
+        some of my work and experience! <br />
+        If you like what you see and have <br />
+        a project you need coded, <br />
+        don’t hestiate to contact me.
+      </p>
+
+      <div class="para-box">
+        <p class="paragraph2">
+          I can help you build a product , feature or website Look through some
+          of my work and <br />
+          experience! If you like what you see and have a project you need
+          coded, <br />
+          don’t hestiate to contact me.
+        </p>
+      </div>
+
+      <div class="icons">
+        <ul class="icon-row">
+          <li>
+            <a href="https://twitter.com/david630"
+              ><img class="tweet" src="twitter.svg" alt="twitter picture"
+            /></a>
+          </li>
+
+          <li>
+            <img class="med" src="medium.svg" alt="medium picture" />
+          </li>
+
+          <li>
+            <a href="https://github.com/david63011"
+              ><img class="git" src="github.svg" alt="git picture"
+            /></a>
+          </li>
+
+          <li>
+            <img class="angel" src="angellist.svg" alt="angelist picture" />
+          </li>
+        </ul>
+      </div>
+      <div class="top-info">
+        <h1 class="works">
+          My Recent <br />
+          Works
+        </h1>
+        <img class="Indicator" src="Indicator.png" alt="green line" />
+      </div>
+    </div>
+
+    <h2 class="works2">My Recent Works</h2>
+
+    <section class="grid-containers">
+      <div class="container">
```

Comment:
- [ ] Kindly ensure that your project card content matches the Figma design template you use 👍🏽. Please add the appropriate margin/padding and remove the dark border on the **See Project** button as well 😄 

Figma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/187988635-9f39ed8d-ffd4-4e19-b348-383f6d71fdc1.png)

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/187988946-60ccb4ec-3ccb-4227-9d5d-14fec7aeffc0.png)

Reply:
ok, I thought I had the correct amount of padding on the left and right but I increased it I hope now it is perfect and I added the correct margin on the bottom and removed the border